### PR TITLE
Improve storage module

### LINKED ISSUE #26: Add features to storage module

### PR DESCRIPTION
**Pull request by staadecker**
This PR adds the following features to the storage module:

- Parameter for discharge efficiency (previously only had charge efficiency/round-trip efficiency)
- Daily self-discharge rate parameter
- Land use parameter
- Land use expression

This PR also includes the script to add these columns to the database and to update the get_inputs.

_Requires merging #53 first_

### CODE REVIEW

File: switch_model/generators/extensions/storage.py [hunk 1]
```diff
@@ -35,6 +36,13 @@ def define_components(mod):
     for extended time perios, then those behaviors will need to be
     modeled in more detail.
 
+    gen_discharge_efficiency[STORAGE_GENS] describes the efficiency during
+    discharging. A discharge efficiency of 0.90 means that 95% of the energy
```
Comment (PatyHidalgo) [inline switch_model/generators/extensions/storage.py]: 90%

File: switch_model/generators/extensions/storage.py [hunk 2]
```diff
@@ -53,6 +61,13 @@ def define_components(mod):
     per year; one cycle is defined as discharging an amount of energy
     equal to the storage capacity of the project.
 
+    gen_self_discharge_rate[STORAGE_GENS] is the fraction of the charge that is lost
+    over a day. This is used for certain types of storage such as thermal energy
+    storage that slowly looses its charge over time. Default is 0 (no self discharge).
```
Comment (PatyHidalgo) [inline switch_model/generators/extensions/storage.py]: loses

**Review by PatyHidalgo — APPROVED** [review]

**Review by PatyHidalgo — CHANGES_REQUESTED** [review]
Two text edits :)

**Comment by staadecker** [comment]
Fixed!

**Review by PatyHidalgo — APPROVED** [review]

**Comment by PatyHidalgo** [comment]
Thank you!

**Pull request merged**